daemon/api_test.go

```diff
@@ -765,19 +769,31 @@ func (c cfgc) Load(string) (snappy.Part, error) {
 	return &tP{name: "foo", version: "v1", origin: "bar", isActive: true, config: c.cfg, configErr: c.err}, nil
 }
 
+type fakeOverlord struct {
+	configs map[string]string
+}
+
+func (o *fakeOverlord) Configure(s *snappy.SnapPart, c []byte) (string, error) {
+	if string(c) != "" {
```

Comment:
Not such a big deal since this is a test, but `len(c) > 0` would avoid the conversion.